src/mediaEditor/plugins/mediaAlignment/tpl/mediaAlignment.tpl

```diff
@@ -0,0 +1,23 @@
+<h3 class="media-sizer-panel-label">{{__ 'Position'}}</h3>
+<div class="media-alignment">
```

Comment:
Inputs can be grouped with fieldset tag, this is more semantical and better for a11y
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset